velero/backup/common-service/label-common-service.sh

```diff
@@ -178,15 +184,41 @@ function label_subscription() {
     ${OC} label subscriptions.operators.coreos.com $cs_pm foundationservices.cloudpak.ibm.com=subscription -n $OPERATOR_NS --overwrite=true 2>/dev/null
     ${OC} label subscriptions.operators.coreos.com $cm_pm foundationservices.cloudpak.ibm.com=singleton-subscription -n $CERT_MANAGER_NAMESPACE --overwrite=true 2>/dev/null
     ${OC} label subscriptions.operators.coreos.com $lis_pm foundationservices.cloudpak.ibm.com=singleton-subscription -n $LICENSING_NAMESPACE --overwrite=true 2>/dev/null
-    ${OC} label subscriptions.operators.coreos.com $lsr_pm foundationservices.cloudpak.ibm.com=singleton-subscription -n $LSR_NAMESPACE --overwrite=true 2>/dev/null
+    ${OC} label subscriptions.operators.coreos.com $lsr_pm foundationservices.cloudpak.ibm.com=lsr -n $LSR_NAMESPACE --overwrite=true 2>/dev/null
+    echo ""
+}
+
+function label_lsr() {
+    
+    title "Start to label the License Service Reporter... "
+    ${OC} label customresourcedefinition ibmlicenseservicereporters.operator.ibm.com foundationservices.cloudpak.ibm.com=lsr --overwrite=true 2>/dev/null
+    ${OC} label ibmlicenseservicereporters.operator.ibm.com ibm-lsr-instance foundationservices.cloudpak.ibm.com=lsr -n $LSR_NAMESPACE --overwrite=true 2>/dev/null
```

Comment:
I created my LSR instance using the licensing team's instance here https://github.ibm.com/cloud-license-reporter/ibm-licensing-ci/blob/master/scripts/tools_ls/ibm_ls_reporter_install_cd_custom.sh#L203 and it's just called "instance". We may need to have a value set as the default like ibm-lsr-instance with the option to change it in the env.properties file